docs/cli.rst

```diff
@@ -48,6 +51,12 @@ Or with a filename::
     export FLASK_APP=/path/to/hello.py
     flask run
 
+If you use Flask 0.13 or later you can also set the environment variables
+in a ``.env`` file.  For instance like this::
+
+    FLASK_APP=/path/to/hello.py
+    FLASK_DEBUG=1
+
```

Comment:
> So this will not work for production use (intentionally).

And you don't think that's odd? I can do `flask db upgrade` just fine on my dev box, but when I got to my production EC2 I have to do `FLASK_APP=app flask db upgrade`?